### PR TITLE
dom-events

### PR DESCRIPTION
**Pull request by John-bess4**
Screen Shot 2023-05-22 at 1 51 13 PM

### CODE REVIEW

**Comment by John-bess4** [comment]
Screen Shot 2023-05-22 at 1 52 08 PM
  
  
  Last image didn't have the Click me output, this one has all three.

File: dom-events/buttons.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+function handleClick(event) {
+  console.log('button clicked');
+  console.log('event:', event);
+  console.log(event.target);
+}
+
+const clickButton = document.querySelector('.click-button');
+clickButton.addEventListener('click', handleClick);
```
Comment (shawnkost) [inline dom-events/buttons.js:8]: Don't forget to include that `$` in front of your variable names! Whenever we store DOM elements to a variable, we want to include that dollar sign

**Review by shawnkost — APPROVED** [review]
Approved, but check out my comment!

File: dom-events/buttons.js (hunk 1 above)
Comment (John-bess4) [reply dom-events/buttons.js:8]: Okay noted, will make sure to remember next time.

**Pull request merged**